### PR TITLE
add lookup_sparse_table_op

### LINKED ISSUE #10046: lookup_sparse_table op to lookup from the sparse table
After #9897 was merged, we need to use a new operator `lookup_sparse_table` on PServer to deal with the `prefetch` request, if the key does not exist in the sparse table, we can return a value with random data.

### PR DESCRIPTION
**Pull request by Yancey1989**
Fixed #10046 
Task #9211

### CODE REVIEW

**Comment by jacquesqiao** [comment]
Why should we add another lookup_table? In my mind, we can reuse the old lookup_table_op.

**Comment by typhoonzero** [comment]
Agree with @jacquesqiao , current `lookup_table_op` can also use `Get` to get value from the table if weight is `SelectedRows`

**Comment by Yancey1989** [comment]
Hi @typhoonzero @jacquesqiao 
Maybe we can't reuse `lookup_table_op`, because of the table variable is a const Input var in it. We need another new op to implement the `Compute`(No Kernel Op) so that we can edit the input var.

File: paddle/fluid/operators/lookup_sparse_table_op.cc [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                   cpu, value.data<float>(), value.numel() * sizeof(float));
+    }
+  }
+};
+
+class LookupSparseTableOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  LookupSparseTableOpMaker(OpProto *proto, OpAttrChecker *op_checker)
+      : framework::OpProtoAndCheckerMaker(proto, op_checker) {
+    AddInput("W",
```
[the remark this answers is not in the document]
Comment (Yancey1989) [reply paddle/fluid/operators/lookup_sparse_table_op.cc]: Done.

**Pull request merged**